ShoppingStore/PromotionType.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * Enum of Promotion type
+ * 
+ * @author Udit Saxena
+ * 
+ */
+public enum PromotionType {
```

Comment:
Promotion type can be of two types only Amount, Promotion
Independent of Order and product promotion